[solutions] Replace AsciiDoc API links (#433)
elastic/docs-content: solutions/search/vector/knn.md

    ```

> The [document `_score`](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-search) is determined by the similarity between the query and document vector. See [`similarity`](https://www.elastic.co/guide/en/elasticsearch/reference/current/dense-vector.html#dense-vector-similarity) for more information on how kNN search scores are computed.

Reviewer: ```suggestion
The document `_score` is a positive 32-bit floating point number used to determine the relevance of the returned document, determined by the similarity between the query and document vector. See [`similarity`](https://www.elastic.co/guide/en/elasticsearch/reference/current/dense-vector.html#dense-vector-similarity) for more information on how kNN search scores are computed.
```

    { "byte-image-vector": [11, 23], "title": "full moon" }
    ```

> 3. Run the search using the [`knn` option](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-search) ensuring the `query_vector` values are integers within the range [-128, 127].

Reviewer: ```suggestion
3. Run the search using the [`knn` option](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-search#operation-search-body-application-json-knn) ensuring the `query_vector` values are integers within the range [-128, 127].
```

    { "image-vector": [1.2, 0.1], "title": "full moon" }
    ```

> 2. Run the search using the [`knn` option](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-search). When searching, the `float` vector is automatically quantized to a `byte` vector.

Reviewer: ```suggestion
2. Run the search using the [`knn` option](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-search#operation-search-body-application-json-knn). When searching, the `float` vector is automatically quantized to a `byte` vector.
```
